packages/dataviews/src/single-selection-checkbox.js

```diff
@@ -29,33 +29,39 @@ export default function SingleSelectionCheckbox( {
 			: __( 'Deselect item' );
 	}
 	return (
-		<CheckboxControl
-			className="dataviews-view-table-selection-checkbox"
-			__nextHasNoMarginBottom
-			checked={ isSelected }
-			label={ selectionLabel }
-			disabled={ disabled }
-			onChange={ () => {
-				if ( ! isSelected ) {
-					onSelectionChange(
-						data.filter( ( _item ) => {
-							const itemId = getItemId?.( _item );
-							return (
-								itemId === id || selection.includes( itemId )
-							);
-						} )
-					);
-				} else {
-					onSelectionChange(
-						data.filter( ( _item ) => {
-							const itemId = getItemId?.( _item );
-							return (
-								itemId !== id && selection.includes( itemId )
-							);
-						} )
-					);
-				}
-			} }
-		/>
+		<Tooltip
+			text={ disabled ? __( 'Bulk actions are not applicable' ) : null }
```

Comment:
Optional copy suggestion - feel free to ignore.

```suggestion
			text={ disabled ? __( 'Bulk actions not available.' ) : null }
```